pkg/logs/warning/warning.go

```diff
@@ -0,0 +1,46 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package warning
```

Comment:
This package needs to be unit-tested, especially the operations on the map since multiple clients on different threads could call `Raise` and `Get` roughly at the same time.
For more details, you can check https://stackoverflow.com/questions/36167200/how-safe-are-golang-maps-for-concurrent-read-write-operations